search-commons/src/main/java/no/unit/nva/search2/common/AggregationFormat.java

```diff
@@ -133,7 +163,7 @@ static final class Constants {
             LICENSE, "/filter/associatedArtifacts/license"
         );
         private static final Map<String, String> facetResourcePaths2 = Map.of(
-            HAS_FILE, "/filter/associatedArtifacts/hasFile",
+            HAS_FILES, "/filter/hasFiles",
```

Comment:
We have now `hasFiles` facet with following values: 

`hasPublicFile`
`hasNoPublicFile`

Reply:
"reusing" hierarchy increase executing speed.

Reply:
```
            nestedBranchBuilder(ASSOCIATED_ARTIFACTS, ASSOCIATED_ARTIFACTS)
                .subAggregation(visibleForNonOwners())
                .subAggregation(license());
```